fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/domain/SinglePrefer.java

```diff
@@ -41,7 +41,13 @@
      * @throws ParseException if parse exception occurred
      */
     public SinglePrefer(final String header) throws ParseException {
-        preferTags.addAll(HttpHeaderReader.readList(PreferTag::new, header));
+        preferTags.addAll(stream(header.split(",")).map(h -> {
+            try {
+                return new PreferTag(h);
+            } catch (final ParseException e) {
+                throw new IllegalArgumentException("Could not parse 'Prefer' header", e);
+            }
+        }).collect(toSet()));
```

Comment:
OK, great.  In that light, I went ahead and killed it at its source in `PreferTag`, getting rid of that nasty checked exception inside the lambda